.github/workflows/lint.yml

```diff
@@ -23,9 +23,5 @@ jobs:
           node-version: 22.x
           cache: "npm"
       - run: npm ci
+      - run: npm run format:check
       - run: npm run lint
-        env:
-          TERM: xterm-256color
-          # Set to the correct color level; 2 is 256 colors
-          # https://github.com/chalk/chalk?tab=readme-ov-file#supportscolor
-          FORCE_COLOR: 2
```

Comment:
this was copied from the tests (where it's relevant for the test runner), but isn't needed here. this job has color support & the linter detects it:
![image](https://github.com/user-attachments/assets/d8f8c300-80b1-461e-a289-190aea7c4560)